src/index.js

```diff
@@ -0,0 +1,139 @@
+import './style.css';
+
+// Retrieve tasks from local storage
+const getTasksFromStorage = () => {
+  const tasksJSON = localStorage.getItem('tasks');
+  return tasksJSON ? JSON.parse(tasksJSON) : [];
+};
+
+// Save tasks to local storage
+const saveTasksToStorage = (tasks) => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+// Edit a task description
+const editTaskDescription = (index, newDescription) => {
+  const tasks = getTasksFromStorage();
+  let renderTasks;
+  const task = tasks.find((task) => task.index === index);
+  if (task) {
+    task.description = newDescription;
+    saveTasksToStorage(tasks);
+    renderTasks();
+  }
+};
+
+// Delete a task
+const deleteTask = (index) => {
+  let renderTasks;
+  let tasks = getTasksFromStorage();
+  tasks = tasks.filter((task) => task.index !== index);
+  tasks.forEach((task, i) => {
+    task.index = i + 1;
+  });
+  saveTasksToStorage(tasks);
+  renderTasks();
+};
```

Comment:
- [ ] It will be great to ensure that the delete method functions as expected. When I delete an item, the app throws an error on the console and nothing seems to happen. However, when I refresh the page, I notice that the item was removed. kindly ensure that when I
![Screenshot 2023-06-16 at 22 05 12](https://github.com/Eccb7/To-do-List/assets/32361963/9e00c1b0-1d76-4375-80e0-f1985849e639)
 click the delete button the item is removed immediately and also ensure that there is no error thrown on the console. ⬇️